### ISSUE TITLE
Gitlab fails to install - FreeNAS-11.3-U5 - Release channel: 11.3-RELEASE

### ISSUE THREAD

**Issue by jdrames** [ix-plugin-hub/iocage-plugin-index#118]
Since upgrading to FreeNAS-11.3-U5 I have been unable to install Gitlab from the Plugins page.

It appears that the install starts fine. However, on completion the output shows the following log messages:
```
Starting redis.
sed: /usr/local/www/gitlab-ce/config/gitlab.yml: No such file or directory
Starting redis.
sed: /usr/local/gitaly/config.toml: No such file or directory
Starting redis.
sed: /usr/local/www/gitlab-ce/config/database.yml: No such file or directory
Starting redis.
chown: /usr/local/www/gitlab-ce: No such file or directory
Starting redis.
cd: /usr/local/www/gitlab-ce: No such file or directory
Starting redis.
cd: /usr/local/www/gitlab-ce: No such file or directory
Starting redis.
cd: /usr/local/www/gitlab-ce: No such file or directory
Starting redis.
cd: /usr/local/www/gitlab-ce: No such file or directory
Starting redis.
cd: /usr/local/www/gitlab-ce: No such file or directory
Starting nginx...
cd: /usr/local/www/gitlab-ce: No such file or directory
Performing sanity check on nginx configuration:
cd: /usr/local/www/gitlab-ce: No such file or directory
```

I can verify that the /usr/local/www/gitlab-ce directory is missing from the jail. 
I did another install after this and while the install process was going on I monitored the www folder in the jail via shell and interestingly at some point the gitlab-ce directory is created but then later in the process it appears it is removed.

**Comment by theozaurus** [comment]
I've run into this as well. I think it may be because package postgres11-server is specified and gitlab-ce. However, the latest gitlab-ce package needs postgres12-server.

**Comment by jdrames** [comment]
Yeah, currently I've moved over to Gitea for the gitrepository and jenkins to the integration development/production pipeline. So I've got a working system currently in place. Perhaps once the FreeNAS(TrueNAS Core) move to ver 12 it'll be fixed. But with the current system I've got setup I might just skip on converting back to Gitlab.

**Comment by theozaurus** [comment]
I'm still struggling to get it working here. Manually installing `pkg install postgresql12-server` worked, but then failed to start with `Undefined symbol "setproctitle_fast@FBSD_1.5"`. I believe this happens because FreeNAS is currently on FreeBSD 11.3 which is EOL. I then installed it via ports, which got postgres working. However, the lastest version of gitlab-ce (13.4.4) complained:

```
Your current database version is too old to be migrated. You should upgrade to GitLab 11.11.0 before moving to this version. Please see https://docs.gitlab.com/ee/policy/maintenance.html#upgrade-recommendations
```

After checking out GitLab to version 11-11-stable (I had to mess around with Gems a lot as some simply wouldn't complile), I eventually was able to run:

```
rake db:migrate RAILS_ENV=production
```

That worked, but switching once more to GitLab 13.4.4 and running the migrations once more failed as various columns and tables were missing. Perhaps I'm being too ambitous by skipping so many versions.

**Comment by fahadshery** [comment]
I was so fedup that I ended up installing everything from scratch in my own Truenas Core 12.0 jail.
[Here](https://gitlab.fechner.net/mfechner/Gitlab-docu/-/blob/master/install/12.2-freebsd.md) is the guide I used. It was perfectly written and well maintained.

**Comment by fulder** [comment]
> I've run into this as well. I think it may be because package postgres11-server is specified and gitlab-ce. However, the latest gitlab-ce package needs postgres12-server.
As a result installing either one will remove the other.

I can +1 this as I was seeing the exectly same behaviour. This is why I synced the server/client version of postgres in: https://github.com/ix-plugin-hub/iocage-plugin-index/pull/185. Should probably work better with a (clean) installation after this.

**Comment by kubo6472** [comment]
Installs, but error 404 not found is present for me on TrueNAS-12.0-U7. May be related
